TeamBuildingGame/Assets/Scripts/ObjectMover.cs

```diff
@@ -5,15 +5,13 @@
 public class ObjectMover : MonoBehaviour
 {
     GameObject client;
-
-    [SerializeField]
-    GameObject gameState;
     GameStateHandler gameStateHandler;
 
     UDPClient clientData;
 
     void Awake() {
-        gameStateHandler = gameState.GetComponent<GameStateHandler>(); 
+        gameStateHandler = GameObject.Find("GameState").GetComponent<GameStateHandler>();
```

Comment:
er der nogen grund til at finde tingene med find i stedet for bare at sætte dem i unity inspectoren?

Reply:
Gamestate bliver lavet i scenen før main så man kan ikke dragge dem over som referencer